dlairflow/util.py

```diff
@@ -10,17 +10,28 @@
 import os
 
 
-def user_scratch():
+def user_scratch(owner):
```

Comment:
Consider renaming `owner` to `user` or  `user_key` since this method doesn't require it to be a DAG owner.

Reply:
Agreed, done.